satpy/etc/readers/li_l2_nc.yaml

```diff
@@ -0,0 +1,193 @@
+reader:
+  name: li_l2_nc
+  description: MTG Lightning Imager L2 reader
+  sensors: [li]
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+
+file_types:
+  li_l2_le_nc:
+    file_reader: !!python/name:satpy.readers.li_l2_nc.LIL2NCFileHandler
+    file_patterns: ['{pflag}_{location_indicator},{data_designator},{mission_prefix}I{spacecraft_id}+LI-2-LE-{subtype}-{coverage}-{subsetting}-{component1}-BODY-{component3}-{purpose}-{format}_{oflag}_{originator}_{processing_time:%Y%m%d%H%M%S}_{facility_or_tool}_{environment}_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{processing_mode}_{special_compression}_{disposition_mode}_{repeat_cycle_in_day:>04d}_{count_in_repeat_cycle:>04d}.nc']
+    file_desc:
+      product_type: '2-LE'
+      search_paths: ['data', 'state/processor']
+      sectors: ['north', 'east', 'south', 'west']
+      sector_variables: ['detector',
+                        'epoch_time',
+                        'time_offset',
+                        'event_id',
+                        'group_id',
+                        'flash_id',
+                        'l1b_chunk_ids',
+                        'l1b_chunk_offsets',
+                        'l1b_window',
+                        'filter_values',
+                        'l1b_filter_qa',
+                        'l2_group_filter_qa',
+                        'l2_flash_filter_qa']
+      variables: []
+      variable_transforms:
+        l1b_chunk_offsets:
+          accumulate_index_offset: "{sector_name}/l1b_window"
+        epoch_time:
+          seconds_to_datetime: "2000-01-01T00:00:00.000000"
```

Comment:
Are these epoch times such that they might change separately for different products, or could it be defined in the code as a single constant?

Reply:
Now defined as a single constant in the code as suggested.